packages/ses/src/compartment-shim.js

```diff
@@ -242,6 +241,7 @@ export const makeCompartmentConstructor = (intrinsics, nativeBrander) => {
     initGlobalObject(globalObject, intrinsics, sharedGlobalPropertyNames, {
       globalTransforms,
       nativeBrander,
+      makeCompartmentConstructor,
     });
 
```

Comment:
Is the comment on lines 321-323 below still accurate?

Reply:
This change certainly doesn’t change the verity of the TODO comment. Will have to look into it further.